apps/frontend/app/HomePage/ImageCard.tsx

```diff
@@ -1,38 +1,45 @@
-"use client";
+"use client"; 
 import Image, { StaticImageData } from "next/image";
- 
-  interface ImageCardProps {
-    className: string;
-    src: StaticImageData;
-    alt: string;
-    label: string;
-    onClickOption: string;
-  }
+import { useRouter } from 'next/navigation';
+import RutasNavegacion from "../../ routes"; 
+
+interface ImageCardProps {
+  className: string;
+  src: StaticImageData;
+  alt: string;
+  label: string;
+  onClickOption: string;
+}
+
+export default function ImageCard({ className, src, alt, label, onClickOption }: ImageCardProps) {
+  const router = useRouter(); 
 
   const handleImageClick = (text: string) => {
```

Comment:
Do this component more flexible...
something like:

export default function ImageCard({ className, src, alt, label, onClickOption }: ImageCardProps) {
  const router = useRouter(); 

  const handleImageClick = () => {
    const route = RutasNavegacion[onClickOption]; 
    if (route) {
      router.push(route);
    } else {
      console.error("Route doesn't exist.");
    }
  };

Reply:
Done